src/sql/workbench/contrib/editData/browser/editDataEditor.ts

```diff
@@ -45,7 +45,7 @@ export class EditDataEditor extends BaseEditor {
 	public static ID: string = 'workbench.editor.editDataEditor';
 
 	// The minimum width/height of the editors hosted in the QueryEditor
-	private readonly _minEditorSize: number = 220;
+	private readonly _minEditorSize: number = 55; //original value was 220 (causes sash to be immobile)
```

Comment:
probably don't need the comment since it refers to old program behavior, which I suspect will largely be irrelevant to future code readers.